src/test/adapter.ts

```diff
@@ -9,7 +9,7 @@ const assert = chai.assert;
 
 describe('PHP Debug Adapter', () => {
 
-    const TEST_PROJECT = path.normalize(__dirname + '/../../testproject');
+    const TEST_PROJECT = path.normalize(__dirname + '/../../testproject/main');
```

Comment:
Some tests use /variables.php which has been moved to /testproject/main/variables.php to test multiple source roots.